packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -456,22 +466,32 @@ export function getValidatorApi({
         chain.executionBuilder !== undefined &&
         builderSelection !== routes.validator.BuilderSelection.ExecutionOnly;
 
-      logger.verbose("Assembling block with produceEngineOrBuilderBlock ", {
+      const loggerContext = {
         fork,
         builderSelection,
         slot,
         isBuilderEnabled,
         strictFeeRecipientCheck,
         // winston logger doesn't like bigint
         builderBoostFactor: `${builderBoostFactor}`,
+      };
+
+      logger.verbose("Assembling block with produceEngineOrBuilderBlock", loggerContext);
+      const commonBlockBody = await chain.produceCommonBlockBody({
+        slot,
+        randaoReveal,
+        graffiti: toGraffitiBuffer(graffiti || ""),
       });
+      logger.debug("Produced phase 0 block body", loggerContext);
```

Comment:
`produceCommonBlockBody` does not only produce phase 0 blocks

Reply:
It was producing only phase0 earlier.